Collaboration draft (#1021)
RSE-Sheffield/RSE-Sheffield.github.io: pages/collaboration/collab-guide.md

---
title: Research collaborations
permalink: /collaboration/collab-guide/
slug: evidence
type: text
---

> As describe in [About the RSE team](TBA), the RSE team at the University of Sheffield is a group of research software specialists with multi-disciplinary knowledge and a wide range of research software skills. With a deep interest in research and software, we are collaborating with all faculties at the University.    

Reviewer: ```suggestion
As described in [About the RSE team](TBA), the RSE team at the University of Sheffield is a group of research software specialists with multi-disciplinary knowledge, a wide range of research software skills and a deep interest in improving research software by collaborating with all faculties at the University.
```

Embedding a RSE in your project is equivalent to initiating a collaboration with a researcher specialising in software engineering. RSEs are research partners, not simply software developers. We have a strong track record of securing funding as Principal Investigators (PIs) and Co-Investigators (Co-Is), as well as co-authoring academic publications. Unlike short-term contract staff, we provide continuity: there is no recruitment process required, and no risk of losing expertise at the end of a project due to project/contract expiry and people leaving the University. In addition, it gives more opportunity to collaborate again together in the future.

> Collaborations span various lenghts, from a few days or a few weeks to long term partnerships of multiple years. However, it is clear that we are much more efficient and impacful when we collaborate on long projects where our contribution to the research itself can be the most visible. 

Reviewer: ```suggestion
Collaborations span various lengths, from a few days or weeks to longer term partnerships of multiple years. However, it is clear that we are much more efficient and impactful when we collaborate on long projects where our contribution to the research itself can be the most visible. 
```

<div id="accordion">
  <div class="card">
    <div class="card-header" id="headingTwo">
      <h5 class="mb-0">
        <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="true" aria-controls="collapseTwo">
          What can you expect from us?
        </button>
      </h5>
    </div>
    <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
      <div class="card-body">

RSEs are not technical support, they are active research collaborators. You should expect your RSE to pro-actively contribute to the research direction and steer software development using open, efficient, and reproducible methodologies. <b>RSEs will always advocate for the formal release of software and the publication of research findings.</b> <br>

> To maintain their high level of expertise and expand their knowledge, RSEs participate in university-wide and external academic activities. Our team culture supports this by prioritizing research impact over rigid time-tracking, providing RSEs with the professional autonomy to balance these development opportunities with project delivery. However, to maintain team cohesion and technical standards, RSEs have the following standing commitments:

Reviewer: Maybe, but not sure...

```suggestion
To maintain their high level of expertise and expand their knowledge, RSEs participate in university-wide and external academic activities. Our team culture supports this by prioritizing research impact over rigid time-tracking, providing RSEs with the professional autonomy to balance their development opportunities with project delivery. However, to maintain team cohesion and technical standards, RSEs have the following standing commitments:
```

<ul>
  <li>Weekly (or fortnightly) one-to-one meetings with their direct Line manager (~30min-1h), </li>

>   <li>Fortnightly meeting with the HoRSE (<30minutes).</li>

Reviewer: Useful to define acronyms people are unlikely to be familiar with.

```suggestion
  <li>Fortnightly meeting with the Head of RSE (<30minutes).</li>
```

  <li>RSE team meetings: 60-minutes RSE group meeting every other Wednesday. </li>
  <li>RSE away day: Once a year, full day. </li> 

>   <li>[Senior RSEs only:] Monthly senior RSE meeting and weekly (or fortnightly) meeting with their report (~30min-1h). </li>

Reviewer: Seems incomplete finishing with `report`?

</ul>

> Depending on their interest and availability, RSEs can also engage in other activitites such as scoping new collaborations, organizing workshops, delivering RSE training, and attending key conferences (such as RSECon).  <b>As you would expect any of your collaborators to also have other activities, you can expect the same from RSEs. However, we will make sure it does not take up more than 10% of their time along the year.</b><br><br>

Reviewer: ```suggestion
Depending on their interest and availability, RSEs can also engage in other activitites such as scoping new collaborations, organizing workshops, delivering RSE training, and attending key conferences (such as the Society of RSE annual conference).  <b>As you would expect any of your collaborators to also have other activities, you can expect the same from RSEs. However, we will make sure it does not take up more than 10% of their time along the year.</b><br><br>
```

<b>Transition period:</b><br>

> Research rarely concludes abruptly at a contract’s end date. Often, essential tasks like finalizing a publication, releasing a software version, or polishing documentation extend slightly beyond the official project timeline. We allow RSEs a reasonable amount of time to finalize these high-impact tasks for previous projects. Equally, when an RSE begins a project with you, they may occasionally need brief windows of time to wrap up their previous collaboration. We appreciate your flexibility in recognizing this as a standard part of the research lifecycle.

Reviewer: ```suggestion
Research rarely concludes abruptly at a contract’s end date. Often, essential tasks like finalizing a publication, releasing a software version, or polishing documentation extend slightly beyond the official project timeline. We allow RSEs a reasonable amount of time to finalize these high-impact tasks for previous projects. Equally, when an RSE begins a project with you, they may occasionally need brief windows of time to wrap up their previous collaboration whilst familiarising themselves with your project. We appreciate your flexibility in recognizing this as a standard part of the research lifecycle.
```

   </div>
  </div>
 </div>

  <div class="card">
    <div class="card-header" id="headingSix">
      <h5 class="mb-0">
        <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseSix" aria-expanded="false" aria-controls="collapseSix">
          What will we expect from you?
        </button>
      </h5>
    </div>
    <div id="collapseSix" class="collapse" aria-labelledby="headingSix" data-parent="#accordion">
      <div class="card-body">

Collaborating with a Research Software Engineer (RSE) is fundamentally similar to working with any academic researcher. To ensure the success of our joint efforts, please keep the following operational principles in mind:

<ul>

> <li> <b>Onboarding</b>: RSEs are versatile individuals who frequently transition between diverse fields, such as moving from particle physics to archaeology. While this cross-disciplinary expertise allows for the valuable transfer of techniques, it requires an good understanding of the project from the beginning. To ensure an efficient start, we require that the PI arrives at the kick-off meeting with all necessary onboarding materials (see below). This can include relevant publications, existing codebases, technical documentation, and domain-specific context.</li> 

Reviewer: ```suggestion
<li> <b>On-boarding</b>: RSEs are versatile individuals who frequently transition between diverse fields, such as moving from particle physics to archaeology. While this cross-disciplinary expertise allows for the valuable transfer of techniques, it requires a good understanding of the project from the beginning. To ensure an efficient start, we require that the PI arrives at the kick-off meeting with all necessary on-boarding materials (see below). This can include relevant publications, existing codebases, technical documentation, and domain-specific context.</li> 
```

> <li> <b>Teamwork and communication:</b> An RSE should be treated as a core member of your research team rather than an external service provider. We expect RSEs to be included in team meetings, relevant mailing lists, and communication channels. Active engagement from the Principal Investigator is critical. Regular meeting regarding the software’s evolution are essential to keep the project on track and scientifically relevant.</li> 

Reviewer: ```suggestion
<li> <b>Teamwork and communication:</b> An RSE should be treated as a core member of your research team rather than an external service provider. We expect RSEs to be included in team meetings, relevant mailing lists, and communication channels. Active engagement from the Principal Investigator is critical. Regular meetings regarding the software’s evolution are essential to keep the project on track and scientifically relevant.</li> 
```

<li> <b>Outputs:</b> Like all research staff, RSEs are evaluated based on their scholarly contributions. <b>As a general rule, If the software developed or maintained by the RSE has enabled the content of a publication co-authorship is expected</b>. We obviously do not request authorship on unrelated works; however, if the RSE’s contributions enabled the research, they should be credited as a collaborator. </li>
</ul>  

      </div>
    </div>
  </div>

</div>

<br>

# In practice

> The RSE team is collaborating on more than 30 projects a year. To make sure that we can have a general overview on how all of them are going, it is important to have a project management process. The goal is not to micro-manage each of them. RSEs on projects are managing the overall RSE work. The objective is to keep track of project and be able to tackle potential blockers that may arise during the collaboration to make sure that all parties have an anjoyable experience.  

Reviewer: ```suggestion
The RSE team collaborates on more than 30 projects a year. To make sure that we can have a general overview on how all of them are going, it is important to have a project management process. The goal is not to micro-manage each of them. RSEs on projects are managing the overall RSE work. The objective is to keep track of progress on the project and be able to tackle potential blockers that may arise during the collaboration to make sure that all parties have an enjoyable experience.  
```

> For this reason, we have created a fairly light process of project management, from scoping to impact inquiry. You can see it on the figure below.  

Reviewer: ```suggestion
For this reason, we have created a fairly light process of project management, from scoping to impact inquiry shown in the figure below.  
```

<img src="/assets/images/2026_RSE_project_timeline.png" alt="Project timeline showing the different stages of a project with RSEs. "/>

In practice, 

<div id="accordion">
  <div class="card">
    <div class="card-header" id="headingTwo">
      <h5 class="mb-0">
        <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="true" aria-controls="collapseTwo">
          Project setup: How to Contact us and how to cost us?
        </button>
      </h5>
    </div>
    <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
      <div class="card-body">
        
       <b>WHEN to contact us?</b><br>

        We strongly prefer to engage with our colleagues as early as possible in the research project timeline, <b>ideally during the grant writing process</b>. This allows for us to:
        <ul>

>           <li>Participate in the shaping of the proposal and advise you on the best software approach for the project. We might, for very software heavy proposal, suggest to join the project as co-I. In that case we will be happy to participate in the writing of the grant.</li>

Reviewer: ```suggestion
          <li>Participate in the shaping of the proposal and advise you on the best software approach for the project. We might, for very software heavy proposals, suggest to join the project as co-investigator. In that case we will be happy to participate in the writing of the grant.</li>
```

          <li>See if we have the right skills in the group,</li>
          <li>Estimate the cost of the RSE involvment (see below),</li> 
          <li>Adjust the allocation for the team in advance.</li>
        </ul>

        We also can collaborate with you at the post-award stage, but we cannot ensure the availabily of the RSEs at that stage.<br><br>

        <b>HOW to contact us?</b><br>

>         To engage with us and discuss a potential collaboration you can write a mail to <a href="mailto:rse@sheffield.ac.uk">rse@sheffield.ac.uk</a> and complete this <a href="https://forms.gle/67NepxJxUGyRYMeu6">form</a>. The form helps us to have an idea of the project and decide who's best to talk to you. We will then get in touch with you and setup a scoping meeting. In general two RSEs will be coming to the meeting. 

Reviewer: How often to do people email and send a form?

I get the impression that a lot of people just email and the content of those emails in relation to the information solicited via the form can be quite variable. 

Would it do any harm to suggest that the form is completed as the primary method of making contact to help extract as much information up front as possible?

The email could be omitted completely or mentioned as an alternative at the end.

```suggestion
        To engage with us and discuss a potential collaboration you please complete this <a href="https://forms.gle/67NepxJxUGyRYMeu6">form</a>. The form helps us to have an idea of the project and decide who from the team is best suited to talk to you. We will then get in touch with you and setup a scoping meeting. In general two RSEs will be attend the scoping meeting. If you have any questions you can always can write email us at href="mailto:rse@sheffield.ac.uk">rse@sheffield.ac.uk</a>.
```

        <br><br>

        <b>How to COST us?</b><br>
        Costing a RSE depends on the source of the funding:

        <ul>

>           <li>If the grant is generating overheads, the RSE is costed as G7.10, as <b>directly incurred</b> cost.</li>

Reviewer: ```suggestion
          <li>If the grant is generating overheads, the RSE is costed as G7.10, as a <b>directly incurred</b> cost.</li>
```

          <li>If the grant is <b>NOT</b> generating overheads, the RSE can be costed using our <i>internal</i> day rate of <b>£384</b>,</li>
          <li>If the funding comes within the University, no overheads are to be taken into account. In that case we use our <i>internal</i> day rate of <b>£384</b>,</li>

>           <li>We are sometimes working for external companies (e.g. spin-outs), in that case the day rate must take into account the overheads is <b>£7XX</b>.</li> 

Reviewer: Amount needs completing, suggested value of £764 is purely my guess based on doubling the standard day-rate of £384
```suggestion
          <li>We are sometimes working for external companies (e.g. spin-outs), in that case the day rate must take into account the overheads and is <b>£768</b>.</li> 
```

Reviewer 2: This is marked as resolved but doesn't appear to be fixed in the PR - is the latest version pushed to github?

          Done? Wrap up meeting
        </button>
      </h5>
    </div>
    <div id="collapseFive" class="collapse" aria-labelledby="headingFive" data-parent="#accordion">
      <div class="card-body">

Ultimately, the goal of the collaborations is to participate to the research projects with robust scientific/technical input. 

It is therefore important to make sure that at the end of the project you are handed over the complete work. Another important aspect of this handover is that you understand the work that is handed over to you. For that reason we encourage the P.I. and/or research group members to take an active part in the development of the software. <br> 
Outputs are specific to projects and vary greatly by programming language, but have some core themes. The RSE team aspires to produce software that can be further developed with minimal or no input from the original engineer. To facilitate this we prioritise including the following in our repositories:
Developer and user documentation: How to build / run the software, how it is organised, what conventions have been used, what developer tools are needed e.g. to run tests.

> Execution environment and dependencies: What operating systems will the software run on, is what other software / libraries / modules are needed, what versions, how the execution environment is described. E.g. through containerisation.

Reviewer: ```suggestion
Execution environment and dependencies: What operating systems will the software run on, what other software / libraries / modules are needed, what versions, how the execution environment is described. E.g. through containerisation.
```

---
title: Research collaborations
permalink: /collaboration/collab-guide/
slug: evidence
type: text
---

The RSE team collaborates on more than 30 projects a year. Collaborations span various lengths, from a few days or a few weeks to longer term partnerships of multiple years. However, it is clear that we are much more efficient and impactful when we collaborate on long projects where our contribution to the research itself can be the most visible. 

> To make sure that we can have a general overview on how all of them are going, it is important to have a project management process. The goal is not to micro-manage each of them. RSEs on projects are managing the overall RSE work. The objective is to keep track of project of the progress on the project and be able to tackle potential blockers that may arise during the collaboration to make sure that all parties have an enjoyable experience.  

Reviewer 2: > how all of them are going

 I would consider changing this to how all of the projects are going as we're in a new paragraph

>The goal is not to micro-manage each of them. RSEs on projects are managing the overall RSE work.

This bit is a bit unclear to me, is "each of them" here referring to each project or each RSE?

> keep track of project of the progress on the project

Typo, extra 'of project'

<div id="rse-accordion" class="accordion shadow-sm"> 
  <div class="card border-bottom-0">
    <div class="card-header" id="headingOne" style="background-color: rgba(90,185,107,0.7); border: none;">
      <h5 class="mb-0">
        <button class="btn btn-link btn-block text-left d-flex justify-content-between align-items-center text-dark font-weight-bold" 
                type="button" 
                data-toggle="collapse" 
                data-target="#collapseOne" 
                aria-expanded="true" 
                aria-controls="collapseOne"
                style="text-decoration: none;">
          Project setup: How to Contact us and how to cost us?
          <i class="fas fa-chevron-down fa-xs text-muted"></i>
        </button>
      </h5>
    </div>
    <div id="collapseOne" class="collapse show" aria-labelledby="headingOne" data-parent="#rse-accordion">
      <div class="card-body bg-light">

       <b>WHEN to contact us?</b><br>

        We strongly prefer to engage with our colleagues as early as possible in the research project timeline, <b>ideally during the grant writing process</b>. This allows for us to:
        <ul>

>           <li>Participate in the shaping of the proposal and advise you on the best software approach for the project. We might, for very software heavy proposals, suggest to join the project as co-investigator. In that case we will be happy to participate in the writing of the grant.</li>

Reviewer 2: ```suggestion
          <li>Participate in the shaping of the proposal and advise you on the best software approach for the project. We might, for very software heavy proposals, suggest suggest that one of our team join the proposal as co-investigator. In such cases we would be happy to participate in the writing of the grant.</li>
```

          <li>See if we have the right skills in the group,</li>
          <li>Estimate the cost of the RSE involvment (see below),</li> 
          <li>Adjust the allocation for the team in advance.</li>

>         </ul>

Reviewer 2: Some of the above bullet points end with commas, some with full stops, perhaps should keep it consistent unless this is intentional?

        We also can collaborate with you at the post-award stage, but we cannot ensure the availabily of the RSEs at that stage.<br><br>

        <b>HOW to contact us?</b><br>

>         To engage with us and discuss a potential collaboration please complete this <a href="https://forms.gle/67NepxJxUGyRYMeu6">form</a>. It helps us to have an idea of the project and decide who from the team is best suited to talk to you. We will then get in touch with you and setup a scoping meeting. In general two RSEs will attend to scoping meeting. If you have any questions you can always email us at <a href="mailto:rse@sheffield.ac.uk">rse@sheffield.ac.uk</a>. 

Reviewer 2: ```suggestion
        To engage with us and discuss a potential collaboration please complete this <a href="https://forms.gle/67NepxJxUGyRYMeu6">form</a>. It helps us to have an idea of the project and decide who from the team is best suited to talk to you. We will then get in touch with you and setup a scoping meeting. In general two RSEs will attend the scoping meeting. If you have any questions you can always email us at <a href="mailto:rse@sheffield.ac.uk">rse@sheffield.ac.uk</a>. 
```

        <br><br>

        <b>How to COST us?</b><br>

>         Costing a RSE depends on the source of the funding:

Reviewer 2: ```suggestion
        Costing an RSE depends on the source of the funding:
```

        <ul>
          <li>If the grant is generating overheads, the RSE is costed as G7.10, as <b>directly incurred</b> cost.</li>
          <li>If the grant is <b>NOT</b> generating overheads, the RSE can be costed using our <i>internal</i> day rate of <b>£384</b>,</li>

>           <li>If the funding comes within the University, no overheads are to be taken into account. In that case we use our <i>internal</i> day rate of <b>£384</b>,</li>

Reviewer 2: ```suggestion
          <li>If the funding comes from within the University, no overheads are to be taken into account. In that case we use our <i>internal</i> day rate of <b>£384</b>,</li>
```

          <li>We are sometimes working for external companies (e.g. spin-outs), in that case the day rate must take into account the overheads is <b>£7XX</b>.</li> 
        </ul>

      </div>
    </div>
  </div>

  <div class="card">
    <div class="card-header" id="headingTwo" style="background-color: rgba(170,194,86,0.7); border: none;">
      <h5 class="mb-0">
        <button class="btn btn-link btn-block text-left d-flex justify-content-between align-items-center text-dark font-weight-bold collapsed" 
                type="button" 
                data-toggle="collapse" 
                data-target="#collapseTwo" 
                aria-expanded="false" 
                aria-controls="collapseTwo"
                style="text-decoration: none;">
          Project Kick-off: Let's work together
          <i class="fas fa-chevron-down fa-xs text-muted"></i>
        </button>
      </h5>
    </div>
    <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#rse-accordion">
      <div class="card-body bg-light">

> When your grant is funded (congratulations!) and that the starting date of the collaboration with the RSE approaches, we will organise a kickoff meeting. This meeting will be attended by the RSE and their line manager and by people from the research team whose work will be closely linked to the RSE’s (at least the P.I. of the project). <b>During the meeting we will discuss the following item:</b> <br><br>

Reviewer 2: > and that the starting date of the collaboration with the RSE approaches

and the starting date of the collaboration with [the assigned RSE | the RSE team] approaches

> discuss the following item

discuss the following items

<ul>
<li><b>Objectives and requirements of the project:</b> What are the overall research objectives of the project?</li>

> <li><b>Configuration of the collaboration:</b> Primary contact, who is going to work on the project, Meeting cadence, Progress meetings,</li>

Reviewer 2: Are capitals for Meeting and Progress intended?

> <li><b>Objectives of the RSE:</b> Mapping research is difficult, so for this item we will discuss what are the objective of the first month?</li>

Reviewer 2: ```suggestion
<li><b>Objectives of the RSE:</b> Mapping research is difficult, so for this item we will discuss and set first-month objectives.</li>
```

<li><b>Risks:</b> What could go wrong about the project?</li>
<li><b>Technical aspects:</b> On-boarding material, issue tracking and communication</li>

> <li><b> Outputs: </b> What outputs do you envisage to produce? (papers, software releases, etc.)</li> 

Reviewer 2: ```suggestion
<li><b> Outputs: </b> What outputs do you envisage producing? (papers, software releases, etc.)</li> 
```

</ul>

> During the meeting, everything that is discussed will be recorded in the RSE <a href="https://docs.google.com/document/d/1niSMu8MPCmZx3CAspb290QCglx-CuY4Ac4LB4AhHLWo/edit?usp=sharing">end-to-end project report</a>. The RSEs working on the project are the owner of the report and will be in charge of keeping it up to date.

Reviewer 2: ```suggestion
During the meeting, everything that is discussed will be recorded in the RSE <a href="https://docs.google.com/document/d/1niSMu8MPCmZx3CAspb290QCglx-CuY4Ac4LB4AhHLWo/edit?usp=sharing">end-to-end project report</a>. The RSEs working on the project are the owners of the report and will be in charge of keeping it up to date.
```

        <button class="btn btn-link btn-block text-left d-flex justify-content-between align-items-center text-dark font-weight-bold collapsed" 
                type="button" 
                data-toggle="collapse" 
                data-target="#collapseThree" 
                aria-expanded="false" 
                aria-controls="collapseThree"
                style="text-decoration: none;">
          Progress meetings: Let's take a step back
          <i class="fas fa-chevron-down fa-xs text-muted"></i>
        </button>
      </h5>
    </div>
    <div id="collapseThree" class="collapse" aria-labelledby="headingThree" data-parent="#rse-accordion">
      <div class="card-body bg-light">
For a long project in particular, it is often useful to take some time along the way to take a step back and discuss how the work and the collaboration is going. This is why we will organise progress meetings (also known as retrospective meetings). It will be an occasion to discuss the following points:<br><br>

 <ul>
          <li>What is going well?</li>
          <li>What is not going so well? </li>
          <li>What can be improved? </li>
          <li>What are the next steps? </li>
</ul>

> It is worth mentioning that we do progress meetings for long projects (6+ months). For shorter project we are doing a final retrospective meetings at the end of the project only. 

Reviewer 2: ```suggestion
It is worth mentioning that we do progress meetings for long projects (6+ months). For shorter projects, we will hold a single retrospective meeting at the end of the project only. 
```

      </div>
    </div>
  </div>

  <div class="card">
    <div class="card-header" id="headingFour" style="background-color: rgba(139,187,133,0.7); border: none;">
      <h5 class="mb-0">
        <button class="btn btn-link btn-block text-left d-flex justify-content-between align-items-center text-dark font-weight-bold collapsed" 
                type="button" 
                data-toggle="collapse" 
                data-target="#collapseFour" 
                aria-expanded="false" 
                aria-controls="collapseFour"
                style="text-decoration: none;">
          Done? Wrap up meeting
          <i class="fas fa-chevron-down fa-xs text-muted"></i>
        </button>
      </h5>
    </div>
    <div id="collapseFour" class="collapse" aria-labelledby="headingFour" data-parent="#rse-accordion">
      <div class="card-body bg-light">

> Ultimately, the goal of the collaborations is to participate to the research projects with robust scientific/technical input. It is therefore important to make sure that at the end of the project you are handed over the complete work. Another important aspect of this handover is that you understand the work that is handed over to you. For that reason we encourage the P.I. and/or research group members to take an active part in the development of the software.<br><br>

Reviewer 2: ```suggestion
Ultimately, the goal of the collaborations is to contribute to the research projects with robust scientific/technical input. It is therefore important to make sure that at the end of the project you are handed over the complete work. Another important aspect of this handover is that you understand the work that is handed over to you. For that reason we encourage the P.I. and/or research group members to take an active part in the development of the software.<br><br>
```

Outputs are specific to projects and vary greatly by programming language, but have some core themes. The RSE team aspires to produce software that can be further developed with minimal or no input from the original engineer. To facilitate this we prioritise including the following in our repositories: 

<ul>
<li> <b>Developer and user documentation</b>: How to build / run the software, how it is organised, what conventions have been used, what developer tools are needed e.g. to run tests. </li>

> <li> <b>Execution environment and dependencies</b>: What operating systems will the software run on, is what other software / libraries / modules are needed, what versions, how the execution environment is described. E.g. through containerisation.</li> 

Reviewer 2: ```suggestion
<li> <b>Execution environment and dependencies</b>: What operating systems will the software run on, what other software / libraries / modules are needed, what versions, how the execution environment is described. E.g. through containerisation.</li> 
```

<li> <b>Appropriate automated tests</b>: unit, regression, integration.</li> 
</ul>
A final retrospective meeting will be held at the end of the project. This will help us to know what went well, what did not, and how to improve our way of working in the future.

      </div>
    </div>
  </div>

  <div class="card">
    <div class="card-header" id="headingFive" style="background-color: rgba(18,169,212,0.7); border: none;">
      <h5 class="mb-0">
        <button class="btn btn-link btn-block text-left d-flex justify-content-between align-items-center text-dark font-weight-bold collapsed" 
                type="button" 
                data-toggle="collapse" 
                data-target="#collapseFive" 
                aria-expanded="false" 
                aria-controls="collapseFive"
                style="text-decoration: none;">
          RSE impact
          <i class="fas fa-chevron-down fa-xs text-muted"></i>
        </button>
      </h5>
    </div>
    <div id="collapseFive" class="collapse" aria-labelledby="headingFive" data-parent="#rse-accordion">
      <div class="card-body bg-light">

> We know that producing outputs takes times and we certainly want to be involved in the production of research outputs. <b>As a general rule, If the software developed or maintained by the RSE has enabled the content of a publication co-authorship is expected. This is valid even if the collaboration has ended. </b> For that reason, you can expect RSEs to help wrap up outputs (e.g. papers, software releases) even after you stopped working with the RSEs. <br> <br> 

Reviewer 2: ```suggestion
We know that producing outputs takes times and we certainly want to be involved in the production of research outputs. <b>As a general rule, if the software developed or maintained by the RSE has enabled the content of a publication, co-authorship is expected. This is valid even if the collaboration has ended. </b> For that reason, you can expect RSEs to help wrap up outputs (e.g. papers, software releases) even after you stopped working with them. <br> <br> 
```

> Finally, we will check with you from time to time if the work made by the RSEs has been used for publication or if there is opportunities for further collaborations.   

Reviewer 2: ```suggestion
Finally, we will check with you from time to time if the work made by the RSEs has been used for publication or if there are opportunities for further collaboration.   
```

      </div>
    </div>
  </div>

</div>

<br>

# Expectations

> As described in [About the RSE team](TBA), the RSE team at the University of Sheffield is a group of research software specialists with multi-disciplinary knowledge, a wide range of research software skills and a deep interest in carrying research with software

Reviewer 2: > carrying research with software

Carrying out research with software?

> Embedding a RSE in your project is equivalent to initiating a collaboration with a researcher specialising in software engineering. RSEs are research partners, not simply software developers. We have a strong track record of securing funding as Principal Investigators (PIs) and Co-Investigators (Co-Is), as well as co-authoring academic publications. Unlike short-term contract staff, we provide continuity: there is no recruitment process required, and no risk of losing expertise at the end of a project due to project/contract expiry and people leaving the University. In addition, it gives more opportunity to collaborate again together in the future.

Reviewer 2: ```suggestion
Embedding an RSE in your project is equivalent to initiating a collaboration with a researcher specialising in software engineering. RSEs are research partners, not simply software developers. We have a strong track record of securing funding as Principal Investigators (PIs) and Co-Investigators (Co-Is), as well as co-authoring academic publications. Unlike short-term contract staff, we provide continuity: there is no recruitment process required, and no risk of losing expertise at the end of a project due to project/contract expiry and people leaving the University. In addition, it gives more opportunity to collaborate again together in the future.
```
